src/main/java/org/mitre/synthea/engine/State.java

```diff
@@ -1462,13 +1414,6 @@ public void processOnce(Person person, long time) {
     @Override
     public VitalSign clone() {
       VitalSign clone = (VitalSign) super.clone();
```

Comment:
Same question re threadExpProcessor. I won't raise this again but if this is a valid change then apply everywhere else it is needed.